### ISSUE TITLE
allow commenting a closed note

### ISSUE THREAD

**Issue by HolgerJeromin** [openstreetmap/openstreetmap-website#864]
Often i want to add a few details to a closed conversation at a osm note.

**Comment by tomhughes** [comment]
Duplicate of #845 opened by, err, you.

**Comment by HolgerJeromin** [comment]
Nope. #845 is about changeset discussions, they are never closed. Only OSM Notes can be closed.
Sorry for using the word discussion :-)

**Comment by kerosin** [comment]
Good about this: anonymous users can still participate, although the Note is already closed (they can't reopen)

On the other hand closed Notes that are older than 7 days are hidden on the main site. So there might be discussions behind closed doors that nobody notices expect the users which already have been  participating (email notification).

**Comment by HolgerJeromin** [comment]
@kerosin are you sure? 
an anonymous user can not comment http://www.openstreetmap.org/note/299623 for example. Logged in you can, but you have to reactivate the note.

**Comment by kerosin** [comment]
Yeah that's what I meant. The idea is good because anonymous users would be able to comment after closing a Note. On the other hand this discussion after 7 days would only be visible for users that already commented (subscribed).

**Comment by HolgerJeromin** [comment]
ok, i interpreted your "anonymous users can still participate" as "Holger is wrong, right now it is possible to comment a closed not". Thanks for your clarification.

**Comment by math1985** [comment]
I also would like the option to comment on closed notes. Very useful for example to thank another user for resolving a note you created.

**Comment by neiljp** [comment]
@math1985 I appreciate this feature could be used for other things, but re thankyou messages, that could be with the separate 'thanks' feature discussed elsewhere?

**Comment by matkoniecz** [comment]
It would be extremely useful - in many cases followup comment is needed. Currently workarounds such as commenting on related (or not changeset) are needed

Or you need to reopen note (generates notification/email)

comment (working as expected)

close note (generates notification/email)

these two extra notifications are annoying
